.babelrc.js

```diff
@@ -77,6 +78,7 @@ module.exports = {
         './lib/core/src/server',
         './lib/node-logger',
         './lib/codemod',
+        './lib/source-loader/src',
```

Comment:
This directory includes client-side files like `preview.js`, we shouldn't apply node target to them

Reply:
I'll extract the server part there

Reply:
#7510